### PR TITLE
Bump PHP to 8.0

### PR DESCRIPTION
**Pull request by flangofas**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -8,16 +8,14 @@ LABEL org.label-schema.schema-version="1.0" \
...
 # Enable Networking
 RUN echo "NETWORKING=yes" > /etc/sysconfig/network
 
 # Install EPEL & REMI
-RUN rpm -Uvh https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm \
-    && rpm -Uvh http://rpms.remirepo.net/enterprise/remi-release-7.rpm \
-    && yum-config-manager --enable epel \
-    && yum-config-manager --enable remi-php${PHP_VERSION}
+RUN yum-config-manager --enable epel \
+  && yum-config-manager --enable remi-php${PHP_VERSION}
```
Comment (flangofas) [inline Dockerfile:18]:
```
Preparing...                          ########################################
	package epel-release-7-14.noarch is already installed
The command '/bin/sh -c rpm -Uvh https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm     && rpm -Uvh http://rpms.remirepo.net/enterprise/remi-release-7.rpm     && yum-config-manager --enable epel     && yum-config-manager --enable remi-php${PHP_VERSION}' returned a non-zero code: 1
```

17 and 18 lines are causing the above error while building therefore I removed them and it works just fine. Can I proceed with the same changes for PHP 8.1?

Comment (georgeconstantinou) [reply Dockerfile:18]: Looks good to me.

**Review by georgeconstantinou — APPROVED** [review]

**Review by dmitri-alleo — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (flangofas) [reply Dockerfile:18]: OK, thanks!

**Comment by flangofas** [comment]
@dmitri-alleo thanks!

**Pull request closed without merging**